daml/Passage.daml

```diff
@@ -0,0 +1,38 @@
+module Passage where
+
+template LocalPassage with
+    id: Int
```

Comment:
There is implicit `ContractId`, why do you need one more here?

Reply:
This is the ID that was defined at the application/DB level, because we're still working in the context of a legacy system. So the `ContractId` won't really be used by the application because the entity is created in the application before it actually reaches anything DAML-related.